src/components/Crossroad/Crossroad.tsx

```diff
@@ -1,40 +1,185 @@
+  {label: 'Kôš', url: 'https://kos.strom.sk'},
+] as const
+const campItems = [
+  {label: 'TMM', url: 'matik/akcie/tmm'},
+  {label: 'Primat', url: 'matik/akcie/primat'},
+] as const
+
+interface NavigationLinkProps {
+  label: string
+  url: string
+}
+
+const NavigationLink: React.FC<NavigationLinkProps> = ({label, url}) => (
+  <Link
+    variant="h1"
+    href={url}
+    invertColors
+    sx={{width: {xs: 'auto', md: '100%'}, pl: 0}}
+    textSx={{width: {xs: 'auto', md: '100%'}}}
+  >
+    {label}
+  </Link>
+)
+
+const SectionHeader: React.FC<{children: React.ReactNode}> = ({children}) => (
+  <Typography variant="h3" color="white" sx={{pt: 3, pb: 2}}>
+    {children}
+  </Typography>
+)
 
 export const Crossroad: React.FC = () => {
   return (
     <Stack alignItems="stretch" height="100vh" sx={{bgcolor: colors.black}}>
-      <Grid container columnSpacing={{xs: 2, md: 3, lg: 4}} mt={'30vh'} flexGrow={1}>
-        <Grid size={{xs: 0, sm: 3}} display={{xs: 'none', sm: 'flex'}} height="70dvh" justifyContent="end">
+      <Grid container columnSpacing={{xs: 1, md: 2, lg: 4}} mt={{xs: '5vh', md: '30vh'}} flexGrow={1}>
+        <Grid
+          size={{xs: 12, md: 3}}
+          height={{xs: '10vh', md: '70vh'}}
+          justifyContent={{xs: 'center', md: 'end'}}
+          display="flex"
+        >
           <Box>
             <Strom width="100%" height="100%" preserveAspectRatio="xMaxYMin" color="white" />
           </Box>
         </Grid>
-        <Grid size={{xs: 12, sm: 9}} height="70dvh" sx={{overflow: 'auto'}}>
-          <Stack gap={{xs: 1, sm: 2}}>
-            {crossroadItems.map(({label, url, variant}) => (
-              <Link key={label} variant={variant} href={url} invertColors>
-                {label}
-              </Link>
+        <Grid
+          size={{xs: 12, md: 9}}
+          container
+          height={{xs: '80vh', md: '70vh'}}
+          pr={{xs: 1, md: 3}}
+          sx={{overflow: 'auto'}}
+        >
+          {/* Mobile view: stacked list */}
+          <Stack
+            gap={1}
+            sx={{
+              display: {xs: 'flex', md: 'none'},
+              width: '100%',
+              alignItems: 'center',
+            }}
+          >
+            <NavigationLink url="/strom/o-nas" label="O združení" />
+            {seminarItems.map((item) => (
+              <NavigationLink key={item.label} url={item.url} label={item.label} />
+            ))}
+            {competitionItems.map((item) => (
+              <NavigationLink key={item.label} url={item.url} label={item.label} />
+            ))}
+            {campItems.map((item) => (
+              <NavigationLink key={item.label} url={item.url} label={item.label} />
             ))}
+            <NavigationLink url="/strom/podporte-nas" label="Podporte nás" />
           </Stack>
+
+          {/* Desktop view: table */}
+          <Box sx={{width: '100%', height: 'fit-content', display: {xs: 'none', md: 'block'}}}>
```

Comment:
rozmyslam, ci to vlastne neni Stack 😄 ci by mohlo fungovat daco taketo. ale necheckoutoval som
```suggestion
          <Stack sx={{display: {xs: 'none', md: 'flex'}}}>
```